### PR TITLE
Add option to print stdout/stderr.

### PR DESCRIPTION
**Pull request by jasssonpet**

### CODE REVIEW

File: lib/iphone-simulator.js [hunk 1]
```diff
@@ -122,12 +129,19 @@ var iPhoneSimulator = (function () {
...
 
-        if (options.stderr) {
-            config("setSimulatedApplicationStdErrPath", $(options.stderr));
-        }
-
-        if (options.stdout) {
-            config("setSimulatedApplicationStdOutPath", $(options.stdout));
+        if (options.logging) {
+            var logPath = path.join(path.dirname(appPath), path.basename(appPath, ".app") + ".stderr");
+            child_process.execSync(util.format("rm -f %s && mkfifo %s", logPath, logPath));
```
Comment (teobugslayer) [inline lib/iphone-simulator.js]: execSync is available only on node 0.12 and io.js but we support node 0.10 as well.

**Comment by jasssonpet** [comment]
@teobugslayer I have replaced the `execSync` function with `exec`.

**Comment by teobugslayer** [comment]
:+1: go team!

**Comment by Fatme** [comment]
:+1:

**Pull request merged**